src/test/java/uk/gov/companieshouse/api/testdata/service/impl/CompanyProfileServiceImplTest.java

```diff
@@ -47,56 +59,143 @@ class CompanyProfileServiceImplTest {
     @InjectMocks
     private CompanyProfileServiceImpl companyProfileService;
 
-    @Test
-    void createEnglandWales() {
-        final Address mockServiceAddress = new Address(
-                "","","","","",""
-        );
-        CompanySpec spec = new CompanySpec();
+    private Address mockServiceAddress;
+    private CompanySpec spec;
+    private CompanyProfile savedProfile;
+    private final ValidatorFactory factory = Validation.buildDefaultValidatorFactory();
+    private  Validator validator;
+
+    @BeforeEach
+    void setUp() {
+        mockServiceAddress = new Address("", "", "", "", "", "");
+        spec = new CompanySpec();
         spec.setCompanyNumber(COMPANY_NUMBER);
+        savedProfile = new CompanyProfile();
+    }
+
+    // Test that a company profile is created with default company type with England and Wales jurisdiction
+    @Test
+    void createCompanyWithoutCompanyTypeAndWithEnglandWales() {
         spec.setJurisdiction(Jurisdiction.ENGLAND_WALES);
+        spec.setCompanyStatus(COMPANY_STATUS_ADMINISTRATION);
+        assertCompanyProfile( COMPANY_STATUS_ADMINISTRATION, Jurisdiction.ENGLAND_WALES.toString(), COMPANY_TYPE_LTD, false);
```

Comment:
Would it be worth referring to `spec`?

```suggestion
        assertCompanyProfile(spec.getCompanyStatus(), spec.getJurisdiction().toString(), COMPANY_TYPE_LTD, false);
```